master/master.cfg

```diff
@@ -545,6 +545,15 @@ def create_win_factory(os, llvm):
                    timeout = 3600,
                    command = ['cmd', '/c',
                               'for', '%i', 'in', '(bin\\Release\\performance*.exe)', 'do', '(%i)']))
+    factory.addStep(
```

Comment:
Are any of the generators not safe to run in parallel with other things? If not, the performance lock should be grabbed in counting mode rather than exclusive mode.